api/database/migrations/2022_02_08_200400_create_experience_tables.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateExperienceTables extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
```

Comment:
I recommend adding soft deletes to all these tables.  Check the api/database/migrations/2021_12_29_183502_create_skill_tables.php files for an example.